.travis.yml

```diff
@@ -1,9 +1,14 @@
 language: python
 sudo: false
+matrix:
+  include:
+    - python: 3.5
+      env: TOXENV=py35
 env:
-    - TOXENV=py26
     - TOXENV=py27
     - TOXENV=py33
+    - TOXENV=py34
+    - TOXENV=pypy
```

Comment:
LGTM, although I've a very slighy preference for the way https://github.com/zopefoundation/zope.viewlet/blob/master/.travis.yml does it:
```yml
language: python
sudo: false
python:
  - 2.7
  - 3.3
  - 3.4
  - 3.5
  - pypy
  - pypy3
install:
  - pip install tox-travis
script:
  - tox
notifications:
    email: false
```